recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -13,8 +13,8 @@
 +option(SSE4 "SSE4 4.1 support" FALSE)
  option(ZSTD "ZSTD support for KTX2 transcoding/encoding" TRUE)
 +option(BASISU_NO_ITERATOR_DEBUG_LEVEL "Change Iterator debug level" FALSE)
-+option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" OFF)
-+option(TRANSCODER_ONLY "Build only transcoder part of basisu library")
++option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" FALSE)
```

Comment:
It's handled by CMake https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html

```suggestion
```